TruckMuncher/MapViewController.swift

```diff
@@ -43,11 +45,16 @@ class MapViewController: UIViewController,
         
         var muncherImage = UIImage(named: "truckmuncher")
         var muncherImageView = UIImageView(image: muncherImage)
-        var navFrame = navigationController?.navigationBar.frame
-        muncherImageView.frame = CGRectMake(navFrame!.midX - 20.0, 0.0, 40.0, 40.0)
-        navigationController?.navigationBar.addSubview(muncherImageView)
+        muncherImageView.frame = CGRectMake(0, 0, 40, 40)
+        muncherImageView.contentMode = UIViewContentMode.ScaleAspectFit
+        navigationItem.titleView = muncherImageView
```

Comment:
Yeah, good call. That's definitely how it should be done. Whoever wrote it the other way should be fired.